packages/wdio-devtools-service/package.json

```diff
@@ -38,7 +38,7 @@
     "@wdio/logger": "6.0.16",
     "chrome-remote-interface": "^0.28.0",
     "core-js": "~3.6.0",
-    "lighthouse": "^5.1.0",
+    "lighthouse": "^6.1.0",
```

Comment:
updating to v6 of Lighthouse is likely to break at a lot of places where we use internal Lighthouse interfaces to calculate the metrics.

Reply:
@christian-bromann I have updated the breaking changes and everything works fine locally. Feel free to review it now.